src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -369,7 +369,7 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
                     />
                     <RootStack.Screen
                         name={NAVIGATORS.ONBOARDING_MODAL_NAVIGATOR}
-                        options={onboardingScreenOptions}
+                        options={screenOptions.fullScreen}
```

Comment:
This is I assume only for testing now, right?

Reply:
I'm not sure about this one, `onboardingScreenOptions` come from here

```ts
const onboardingScreenOptions = useMemo(() => screenOptions.onboardingModalNavigator(shouldUseNarrowLayout), [screenOptions, shouldUseNarrowLayout]);
```